scripts/ci/run-jenkins.sh

```diff
@@ -308,12 +308,9 @@ if [[ ${CI_TAGS} == *'webassembly'* ]] || [[ ${CI_TAGS} == *'wasm'* ]];
             ${TESTCMD} --label=sm-mini-test --timeout=20m $gnumake -C sdks/wasm run-sm-mini
             ${TESTCMD} --label=jsc-mini-test --timeout=20m $gnumake -C sdks/wasm run-jsc-mini
             #The following tests are not passing yet, so enabling them would make us perma-red
-            #${TESTCMD} --label=mini-corlib --timeout=20m $gnu$gnumake -C sdks/wasm run-all-corlib
+            ${TESTCMD} --label=mini-corlib --timeout=60m $gnu$gnumake -C sdks/wasm run-all-corlib
```

Comment:
```suggestion
            ${TESTCMD} --label=corlib --timeout=60m $gnu$gnumake -C sdks/wasm run-all-corlib
```

since it's not related to `mini`

Reply:
We can also remove `v8-corlib` from line 318 below since it is already included here.

Reply:
done